src/JsonSchema/stories/JsonSchema.stories.js

```diff
@@ -30,7 +31,16 @@ export const Default = ({ schemaKey }) => {
   return <JsonSchema {...props} />
 }
 
-export const String = ({ schemaKey }) => {
+export const PassedAsString = ({ schemaKey }) => {
   const props = { schema: JSON.stringify(schemas[schemaKey]) }
   return <JsonSchema {...props} />
 }
+
+export const AsCodeComponent = ({ schemaKey }) => {
+  const props = {
+    metastring: `{schema: true}`
```

Comment:
idea: if we used an enum rather than a key-value (string:boolean) pair, then logic Code could be simplified by using a switch and allowing for further extensions to be supported more easily.

Reply:
Yeah, I'm open to the meta data being passed into the code component in a different way. It would be a breaking change so we'd want to make sure that a bunch of our client projects are not already using this syntax.